### PR TITLE
Create comments table

### PR DESCRIPTION
**Pull request by MiBBy-4**

### CODE REVIEW

File: app/models/article.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 class Article < ApplicationRecord
   validates :link, presence: true, length: { maximum: 500 }, url: { accept_array: true }
+  validates :title, length: { minimum:3 }
```
Comment (AndreyAnischik) [inline app/models/article.rb:3]: add space please `{ minimum: 3 }`

File: db/migrate/20220112074341_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments do |t|
+      t.text :comment
```
Comment (AndreyAnischik) [inline db/migrate/20220112074341_create_comments.rb:4]: rename this field to `content` or something like this

File: db/migrate/20220112080443_create_article_comments.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateArticleComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :article_comments do |t|
```
Comment (AndreyAnischik) [inline db/migrate/20220112080443_create_article_comments.rb:3]: why did you decide to use many-to-many association?

**Review by AndreyAnischik — CHANGES_REQUESTED** [review]

File: db/migrate/20220112080443_create_article_comments.rb (hunk 3 above)
Comment (MiBBy-4) [reply db/migrate/20220112080443_create_article_comments.rb:3]: So, I think that it was my mistake, cause in this case the most appropriate association is one to many, where we can link one article with many comments. Sorry, my bad. I will change it.

File: db/migrate/20220113103637_rename_comments_column_to_content.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class RenameCommentsColumnToContent < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :comments, :comment, :content
+    #Ex:- rename_column("admin_users", "pasword","hashed_pasword")
```
Comment (AndreyAnischik) [inline db/migrate/20220113103637_rename_comments_column_to_content.rb:4]: what is it? :)

**Review by AndreyAnischik — CHANGES_REQUESTED** [review]

File: db/migrate/20220113103637_rename_comments_column_to_content.rb (hunk 4 above)
Comment (MiBBy-4) [reply db/migrate/20220113103637_rename_comments_column_to_content.rb:4]: > 

I renamed `comment` column in `comments` table to `content`

File: app/models/comment.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  validates :comment
+  validates :username
+
+  has_one :article
```
Comment (AndreyAnischik) [inline app/models/comment.rb:5]: `belongs_to`

File: db/migrate/20220112074341_create_comments.rb (hunk 2 above)
Comment (AndreyAnischik) [inline db/migrate/20220112074341_create_comments.rb:4]: there should be reference to articles table

**Review by AndreyAnischik — CHANGES_REQUESTED** [review]

File: db/migrate/20220113103637_rename_comments_column_to_content.rb (hunk 4 above)
Comment (MiBBy-4) [reply db/migrate/20220113103637_rename_comments_column_to_content.rb:4]: I just realized what you mean. It  is VS Code comment. I will clear it in next commit.

File: app/models/comment.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  validates :comment
```
Comment (AndreyAnischik) [inline app/models/comment.rb:2]: can you please specify options what it validates? also there is no more comment field for comment model.

File: app/models/comment.rb [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+  validates :username
```
Comment (AndreyAnischik) [inline app/models/comment.rb:3]: can you please specify options what it validates?

**Review by AndreyAnischik — CHANGES_REQUESTED** [review]

File: app/models/comment.rb (hunk 6 above)
Comment (MiBBy-4) [reply app/models/comment.rb:2]: Yeah, I forgot about it. Check please next commit.

File: app/models/comment.rb (hunk 7 above)
Comment (MiBBy-4) [reply app/models/comment.rb]: Done. Check please next commit. And I deleted `username` field, cause I think it is useless for analyze system

Comment (MiBBy-4) [reply app/models/comment.rb]: And I also added validates association for comment model with article model

**Review by AndreyAnischik — APPROVED** [review]

**Pull request merged**